### DISCUSSION TITLE
Help a Newbie with a simple issue

### DISCUSSION THREAD

**Discussion by adamcummins123** [elementor/elementor#24826] [Developers Community]
### Start a discussion

Hi Guys, 

I am trying to change the colour of one specific sub-menu background and I have looked through other forums and found the below css:

.menu-item .menu-item-type-post_type .menu-item-object-post .menu-item-645 { color: red; }

Does this look right to change the submenu item for Newton Transport Plan (Actual Maps) > Overall Rail and Luas Map

The website is acratu.com and I am trying to change the background colour of the menu item to red. 

I appreciate any advice

**Comment by m1ga** [comment]
background color is `background-color` not color.

![Screenshot_20240117_220808](https://github.com/elementor/elementor/assets/4334997/ce5c4246-3d62-48c0-a060-34aa34860aa8)

**Reply by adamcummins123** [reply]
Thank you so much I will give this a try today
